src/test/airtable/test_airtable_current_round.test.js

```diff
@@ -27,7 +27,7 @@ beforeEach(async function () {
         'Voting Starts': 'July 8, 2021 23:59',
         'Voting Ends': 'July 12, 2021 23:59',
         'Earmark Percentage': 0.35,
-        Earmarks: `{"${Earmarks.NEW_GENERAL}":{"OCEAN":30000, "USD":28000}, "${Earmarks.NEW_OUTREACH}":{"OCEAN":40000, "USD":38000}, "${Earmarks.CORE_TECH}":{"OCEAN":50000, "USD":48000}}`,
+        Earmarks: `{"${Earmarks.GENERAL}":{"OCEAN":30000, "USD":28000}, "${Earmarks.NEW_OUTREACH}":{"OCEAN":40000, "USD":38000}, "${Earmarks.CORE_TECH}":{"OCEAN":50000, "USD":48000}, "${Earmarks.GRANT_2ND3RD}":{"OCEAN":16000, "USD":8000}}`,
```

Comment:
I think NEW_GENERAL was supposed to equal NEW_ENTRANTS, but that won't make an impact.
Thanks for checking against GRANT_2ND3RD